tooling/Microsoft.VisualStudio.BlazorExtension/Microsoft.VisualStudio.BlazorExtension.csproj

```diff
@@ -99,18 +99,6 @@
       this problem, we're only using the P2P references to force build ordering. Then we include the build
       output of those projects as content.
     -->
-    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.Blazor.Razor.Extensions\Microsoft.AspNetCore.Blazor.Razor.Extensions.csproj">
-      <Name>Microsoft.AspNetCore.Blazor.Razor.Extensions</Name>
-      <Private>False</Private>
-      <IncludeOutputGroupsInVSIX></IncludeOutputGroupsInVSIX>
-      <IncludeOutputGroupsInVSIXLocalOnly></IncludeOutputGroupsInVSIXLocalOnly>
-    </ProjectReference>
-    <Content Include="..\..\src\Microsoft.AspNetCore.Blazor.Razor.Extensions\bin\$(Configuration)\net461\Microsoft.AspNetCore.Blazor.Razor.Extensions.dll">
```

Comment:
Could we leave this one in? If I have time to do the in-product survey this week, it probably would be better to put it into the VSIX assembly rather than language service assembly. I think currently it doesn't have much other than a VSPackage (and I would need a package for the in-product survey).

Reply:
`Microsoft.AspNetCore.Blazor.Razor.Extensions` is an assembly that contains Blazor's customizations for Razor codegen, it's not the thing you're worried about 👍